loadgen/contract/agent-create-vault.js

```diff
@@ -25,25 +25,26 @@ async function getPurseBalance(purse) {
  *
  * @param {startParam} param
  * @typedef {Awaited<ReturnType<typeof startAgent>>} Agent
- * @typedef { Pick<import('../types').NatAssetKit, 'brand' | 'purse' | 'displayInfo' | 'name'>} AssetKit
+ * @typedef { Pick<import('../types').NatAssetKit, 'brand' | 'purse' | 'displayInfo' | 'symbol'>} AssetKit
```

Comment:
Technically, this is not a `symbol`, it's a Zoe `keyword`.  Unfortunately, this change has further repercussions below.

```suggestion
 * @typedef { Pick<import('../types').NatAssetKit, 'brand' | 'purse' | 'displayInfo' | 'keyword'>} AssetKit
```